main.py

```diff
@@ -672,7 +671,7 @@ def escape_appscript(text: str) -> str:
             )
         ]
     )
-    create_line += '].forEach(s=>f.addScaleItem().setTitle(s).setBounds({},{}).setLabels("{}","{}"))'.format(
+    create_line += '].forEach((s,i)=>f.addScaleItem().setTitle(i+1+". "+s).setBounds({},{}).setLabels("{}","{}"))'.format(
```

Comment:
`${i+1}. ${s}` versus `i+1+". "+s` may be easier to read.

```suggestion
    create_line += '].forEach((s,i)=>f.addScaleItem().setTitle(`${i+1}. ${s}`).setBounds({},{}).setLabels("{}","{}"))'.format(
```

Reply:
I get an error running this, and it also doesn't seem like it would generate shorter Apps Script code, or that readability matters since we're trying to minimize character count. I also have hardly used JavaScript at all, so I'm not sure if there's some confusion with what's Python code vs what's Apps Script code or something.

Reply:
`${x}` within backticks are how you do parameter substitution in a string in js. But yes, it helps but readability though thats not the goal here :D